nixos/modules/services/networking/godns.nix

```
@@ -0,0 +1,87 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib)
+    mkEnableOption
+    mkIf
+    mkOption
+    mkPackageOption
+    types
+    ;
+
+  cfg = config.services.godns;
+
+  configFormat = pkgs.formats.yaml { };
```

For consistency with other modules.
```suggestion
  settingsFormat = pkgs.formats.yaml { };
```